oop/js/FizzBuzz.js

```diff
@@ -11,19 +11,18 @@ export class FizzBuzz {
     this.sequence = parseNumber(sequence);
   }
 
-  generate() {
+  generateFizzBuzz(sequence = this.sequence) {
     let result = [];
 
-    for (let index = 1; index <= this.sequence; index++) {
-      if (index % 4 === 0 || index % 7 === 0) {
+    for (let index = 1; index <= sequence; index++) {
+      if (index % 4 == 0 || index % 7 == 0) {
         result.push("fizz buzz");
-      } else if (index % 2 === 1) {
+      } else if (index % 2 == 1) {
         result.push("fizz");
-      } else if (index % 2 === 0) {
+      } else if (index % 2 == 0) {
         result.push("buzz");
       }
     }
-
     return result;
   }
```

Comment:
```sequence``` parameter on this method is strongly unnecessary, as the class itself already has the "sequence" property.

Reply:
This class is already done, so there is no changes needed here.